server/src/main/java/io/crate/expression/reference/sys/node/NodeStatsContext.java

```diff
@@ -111,6 +122,17 @@ public String name() {
         return name;
     }
 
+    public List<String> roles() {
+        return roles.stream()
+            .map(role -> role.roleName().equals("master") ? "master_eligible" : role.roleName())
```

Comment:
I think it's more clear to use `master_eligible` instead of `master` here, to not confuse users with the `is_master` flag.